obelisk/cpp/obelisk_cpp/include/obelisk_controller.h

```diff
@@ -0,0 +1,101 @@
+#pragma once
+
+#include "obelisk_node.h"
+
+namespace obelisk {
+
+    template <typename ControlMessageT, typename EstimatorMessageT> class ObeliskController : public ObeliskNode {
+      public:
+        explicit ObeliskController(const std::string& name, const std::string& ctrl_key = "publisher_ctrl",
```

Comment:
didn't we decide to change the names of these default keys?

Reply:
I actually decided to stick with the python impl default keys, I guess I don't feel too strongly, and these are reasonable.